src/languages/handlebars.js

```diff
@@ -5,19 +5,44 @@ Author: Robin Ward <robin.ward@gmail.com>
 Description: Matcher for Handlebars as well as EmberJS additions.
 Category: template
 */
-function(hljs) {
-  var BUILT_INS = {'builtin-name': 'each in with if else unless bindattr action collection debugger log outlet template unbound view yield lookup'};
-  function MUSTACHE_CONTENTS() {
-    return  {
-      className: 'name',
-      begin: /[a-zA-Z\.-]+/,
+function handlebars(hljs) {
```

Comment:
Lets not change the intro, some of the existing build process is very picky about this... so just `function(hljs)` please.

Reply:
What I like about functions, is that you can write them top-down in the order in which you would read the code. 

```js
function main() {
  if(isThis()) {
    doThat()
  }   
}

function isThis() {...}

function doThat() {...}
```

Mostly, functions are used the other way around, 

```js
function isThis() {...}

function doThat() {...}

function main() {
  if(isThis()) {
    doThat()
  }   
}
```
but I think it makes the code less readable, because when you start reading at the top, you get all the helper functions first and come to the actual calling code last.

But in that case, I actually started replace some functions by variables, and had all tests failing. I can try some more if you like.